mkt/reviewers/urls.py

```diff
@@ -36,7 +36,13 @@
         name='reviewers.apps.review.attachment'),
     url(r'^apps/review/%s/abuse$' % mkt.APP_SLUG, views.app_abuse,
         name='reviewers.apps.review.abuse'),
+
     url(r'^apps/logs$', views.logs, name='reviewers.apps.logs'),
+    url(r'^apps/moderatelogs$', views.moderatelog,
```

Comment:
@andymckay 'apps/reviewlogs' and 'apps/moderatelogs'?  I thought having /moderate under /logs would imply it was a log of moderation of reviewer reviews, rather than moderation of user ratings.  ( /apps/logs/review and /apps/logs/moderate is a possibility too but would have to redirect /logs to something.)